### DISCUSSION TITLE
Markdown Text Editor Integration

### DISCUSSION THREAD

**Discussion by NdR91** [reorproject/reor#435] [Ideas]
First of all I want to tank you for this great app. I love it.

I would like to request the integration of a markdown text editor in Reor to enhance text formatting options. Markdown support would enable users to format notes more flexibly, allowing for headings, bold and italic text, bullet points, code blocks, and more.

Key benefits:

- Improved text formatting flexibility (e.g., headings, lists, links, images, code blocks)
- Enhanced readability and organization of notes
- Popular feature among note-taking apps and familiar to many users

**Comment by BradKML** [comment]
Which test editor would you pick as the gold standard? Some ideas on where to get the UX code:
- https://github.com/marktext/marktext
- https://github.com/benweet/stackedit
- https://github.com/sparksuite/simplemde-markdown-editor
- https://github.com/Milkdown/milkdown
